airflow/gcp/example_dags/example_kubernetes_engine.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG for Google Kubernetes Engine.
+"""
+
+import os
+import airflow
+from airflow import models
+from airflow.gcp.operators.kubernetes_engine import (
+    GKEClusterCreateOperator,
+    GKEClusterDeleteOperator,
+    GKEPodOperator,
+)
+
+GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "example-project")
+GCP_LOCATION = os.environ.get("GCP_GKE_LOCATION", "europe-north1-a")
+
+CLUSTER_NAME = "test-cluster"
```

Comment:
This should be read from environment variables to ensure isolation between tests on CI.

Reply:
I fixed it.